src/libraries/System.Reflection.MetadataLoadContext/src/System/Reflection/TypeLoading/Assemblies/RoAssembly.cs

```diff
@@ -145,6 +145,9 @@ public sealed override AssemblyName[] GetReferencedAssemblies()
 
         // Miscellaneous properties
         public sealed override bool ReflectionOnly => true;
+#if !FEATURE_GAC
```

Comment:
Can this be applied unconditionally? (Same as my other comment.)

Reply:
I think this one needs to remain conditional because we build this for `netcoreapp3.0` and `netstandard2.0` too.

@steveharter - Can you confirm that I need this `#ifdef` to apply `[Obsolete]` to `RoAssembly.GlobalAssemblyCache` only in `net5.0`+ only?

Reply:
I see no reason to have the condition. MetadataLoadContext doesn't provide a way to "resolve" an assembly from the GAC (even if running under .NET Framework) so this will always be `false`.

Reply:
Thanks. I ended up finding it does still need to be conditional, since the DiagnosticId and UrlFormat properties are new in net5.0 and this compiles against lower versions.  I'm going to change the logic to be `#if GAC_OBSOLETIONS` though, so it connotes the reasoning more clearly.